src/ripple/consensus/Consensus.h

```diff
@@ -1635,15 +1633,15 @@ Consensus<Adaptor>::createDisputes(TxSet_t const& o)
 
     int dc = 0;
 
-    for (auto& id : differences)
+    for (auto& [txId, inThisSet] : differences)
```

Comment:
auto const&

Reply:
Fixed in a225b62e5 `Add const to some structured bindings`